stories/Organism/Blocks/ContentCardWithAndWithoutImage/contentcardwithandwithoutimage.scss

```diff
@@ -1,8 +1,42 @@
 @import '../../../assets/scss/variables';
 @import '../../../assets/scss/mixins';
 @import '../../../assets/scss/breakpoints';
-@import '../../../Templates/PublicationPage/publicationpage.scss';
 
-.card-yellow.green{
-    background-color: $color-light-green !important;
-}   
+.content-data {
+    .content-card {
+        margin-bottom: $spacing-16;
+        word-break: break-word;
+
+        &__caption {
+            padding-bottom: $spacing-16;
+        }
+
+        &__image{
+            height: 100%;
+            display: flex;
+            flex-direction: column;
+            justify-content: space-between;
+        }
+
+        &.card,
+        &.card-yellow {
+            &.yellow,
+            &.red,
+            &.green,
+            &.blue,
+            &.gray {
```

Comment:
@neelesh-tippe this will create styles like `.content-card.card-yellow.yellow`, `.content-card.card-yellow.red`, `.content-card.card-yellow.green` and `.content-card.card-yellow.blue`, etc.. Can we remove `card-yellow` and just stick to `.card` class as mentioned in the previous comment?